### ISSUE TITLE
Items overridden by the screen size

### ISSUE THREAD

**Issue by NathanPB** [jsalatas/plasma-pstate#39]
I feel like there is more things bellow the "Nvidia Settings" header, but my laptop screen size can't fit it all. May a scrollbar be a good idea?

![image](https://user-images.githubusercontent.com/18128642/67122404-22214300-f1c4-11e9-8b9c-c0a44d4e820f.png)

**Comment by jsalatas** [comment]
Yeah! I plan for some time now to create a more compact layout that can also fit in the systray's size. 

Hope I'll find some time to do it soon :\

In the mean time I guess that there are no additional settings below the nvidia section, unless  you have a Dell or an LG computer.
